tests/sentry/issues/test_issue_velocity.py

```diff
@@ -150,9 +153,7 @@ def test_get_latest_threshold_simple(self, mock_update):
         """
         redis_client = get_redis_client()
         redis_client.set(THRESHOLD_KEY.format(project_id=self.project.id), 0.1)
-        redis_client.set(
-            STALE_DATE_KEY.format(project_id=self.project.id), convert_date_to_int(self.utcnow)
-        )
+        redis_client.set(STALE_DATE_KEY.format(project_id=self.project.id), str(self.utcnow))
```

Comment:
What happens if old code was to pick up this new format, let's say during canary deploy?